mmv1/products/monitoring/AlertPolicy.yaml

```diff
@@ -985,3 +985,10 @@ properties:
         description: |
           The format of the content field. Presently, only the value
           "text/markdown" is supported.
+      - !ruby/object:Api::Type::String
+        name: subject
```

Comment:
We can probably expand the at_least_one_of to include this field as well. We generally use that to make sure users can't set empty objects because they can cause problems.

Can you modify the above at_least_one_of blocks to add 
`documentation.0.subject` and then add that block to the subject field as well?

Reply:
Done.